libs/ledger-live-common/src/families/polkadot/setup.ts

```diff
@@ -1,19 +1,30 @@
 // Goal of this file is to inject all necessary device/signer dependency to coin-modules
 
 import { createBridges, type Transaction } from "@ledgerhq/coin-polkadot";
+import { PolkadotCoinConfig } from "@ledgerhq/coin-polkadot/config";
 import polkadotResolver from "@ledgerhq/coin-polkadot/signer/index";
 import makeCliTools, { type CliTools } from "@ledgerhq/coin-polkadot/test/cli";
 import Polkadot from "@ledgerhq/hw-app-polkadot";
 import Transport from "@ledgerhq/hw-transport";
 import type { Bridge } from "@ledgerhq/types-live";
 import { CreateSigner, createResolver, executeWithSigner } from "../../bridge/setup";
 import { Resolver } from "../../hw/getAddress/types";
+import { getCurrencyConfiguration } from "../../config";
+import { getCryptoCurrencyById } from "@ledgerhq/cryptoassets/currencies";
 
 const createSigner: CreateSigner<Polkadot> = (transport: Transport) => {
   return new Polkadot(transport);
 };
 
-const bridge: Bridge<Transaction> = createBridges(executeWithSigner(createSigner));
+const getCurrencyConfig = (): PolkadotCoinConfig => {
+  const polkadot = getCryptoCurrencyById("polkadot");
```

Comment:
We could extract this call outside of the function, so the result is cached during all the Ledger Live lifecycle (and avoid is computation each call).